nengo_viz/server.py

```diff
@@ -11,24 +11,38 @@
     from urllib.parse import unquote
 
 import nengo_viz.swi as swi
+import nengo_viz
 
 
 class Server(swi.SimpleWebInterface):
     """Web server interface to nengo_viz"""
 
     def swi_browse(self, dir):
-        self.script_path = '.'
         if self.user is None: return
         r = ['<ul class="jqueryFileTree" style="display: none;">']
         d = unquote(dir)
-        for f in sorted(os.listdir(os.path.join(self.script_path, d))):
-            ff = os.path.relpath(os.path.join(self.script_path, d,f), self.script_path)
-            if os.path.isdir(os.path.join(self.script_path, d, ff)):
-                r.append('<li class="directory collapsed"><a href="#" rel="%s/">%s</a></li>' % (ff,f))
+        ex_tag = '//examples//'
```

Comment:
I'm not sure if this is the intent, but for me this shows up literally `//examples//` in the file browser. It looks super weird... I might expect something more like a bold or italics on the folder name?

I think at some point (probably not in this PR) we should be making a more stark distinction between user generated scripts (which we can be reasonably sure are in some place that the user can modify) and our provided examples (which are likely to be in a system directory, and therefore are readonly). Something like a separate dropdown menu for examples, rather than having it be part of the same file tree browser.